include/ebpf_program_types.h

```diff
@@ -55,6 +57,7 @@ typedef struct _ebpf_program_data
 {
     ebpf_program_info_t* program_info;
     ebpf_helper_function_addresses_t* helper_function_addresses;
```

Comment:
```suggestion
    ebpf_helper_function_addresses_t* program_type_specific_helper_function_addresses;
```

Reply:
Changed to `program_specific_helper_function_addresses`